code/__DEFINES/crafting.dm

```diff
@@ -26,16 +26,16 @@
 #define CRAFT_ON_SOLID_GROUND (1<<4)
 /// If the craft checks that there are objects with density in the same turf when being built
 #define CRAFT_CHECK_DENSITY (1<<5)
-/// If the created atom will gain custom mat datums
-#define CRAFT_APPLIES_MATS (1<<6)
 /// Crafting passes reagents of components to the finished product
-#define CRAFT_TRANSFERS_REAGENTS (1<<7)
+#define CRAFT_TRANSFERS_REAGENTS (1<<6)
 /// Crafting clears all reagents present in the finished product
-#define CRAFT_CLEARS_REAGENTS (1<<8)
-/// For the crafting unit test, ensures that the custom materials of an item are the same when crafted and spawned.
-#define CRAFT_ENFORCE_MATERIALS_PARITY (1<<9)
+#define CRAFT_CLEARS_REAGENTS (1<<7)
+/// For the crafting unit test, we don't check if the custom materials of an item are the same when crafted and spawned should its recipe have this flag.
+#define CRAFT_SKIP_MATERIALS_PARITY (1<<8)
```

Comment:
Discussed on discord but there's some granularity here for an additional Bitflag to label crafting recipes whose materials are not necessarily equivalent to what went in, but instead just checks for <= the same amount. The baseball bat is an excellent example of this, as it's crafting recipe calls for 5 sheets of wood, for an item that only has 3.5 sheets of materials by the end, but practically it makes sense that some would be lost in the crafting process.

Reply:
Also told them on Discord that this idea is bad: If we want to make sure some materials are lost in the process, a bitflag sure won't tell us how much of each mat to remove from the total. Instead, I've implemented a list var, which will do a better job at it.